api/envoy/config/filter/http/adaptive_concurrency/v2alpha/BUILD

```diff
@@ -1,15 +0,0 @@
-load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library_internal", "api_proto_package")
```

Comment:
Please don't remove any of the v2alpha protos. Instead, make all your changes in v2alpha, and then you can run the `clone.sh` script or just do nothing and @htuch will run it in his next pass. Sorry, this transition period is confusing.

Basically for now don't worry about v3 and just continue to work in v2alpha.